src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+
+#if !JCW_ONLY_TYPE_NAMES
+using Android.Content.PM;
+using Android.Views;
+#endif  // !JCW_ONLY_TYPE_NAMES
+
+namespace Android.App
+{
+
+	[Serializable]
+	[AttributeUsage (AttributeTargets.Class, 
+			AllowMultiple=false, 
+			Inherited=false)]
+#if !JCW_ONLY_TYPE_NAMES
```

Comment:
I removed those JCW_ONLY_TYPE_NAMES from Android.~/~.cs, while leaving them in Android.Runtime and Java.Interop because at this state it seems quite unwise to make extraneous differences between the corresponding sources in Java.Interop.NamingCustomAttributes and those in this new shared project. Those duplicates could be resolved later, not within this change.